03-client-api/references/answer.yml

```diff
@@ -112,6 +112,22 @@ types:
           returns:
             - "[`Concept`](../concept-api/concept?tab=javascript#concept-methods)"
 
+      - method:
+        common: &method-toJSON
+          title: Retrieve a concept map as a JSON
+          description: Produces a concept map as a JSON, where keys are variable names and values are concepts.
+          returns:
+            - JSON
+        java:
+          <<: *method-toJSON
+          method: concept_map.toJSON();
+        javascript:
+          <<: *method-toJSON
```

Comment:
There's actually a follow up PR to the initial JSON implementation here: https://github.com/vaticle/typedb-client-nodejs/pull/220

which renames it from `toJSON` to `toJSONRecord`

Reply:
Fixed in https://github.com/vaticle/docs/pull/678/commits/7eb27634059a4ee8c9f38ab9eed9e2b0345f72c4